gitea/repo_collaborator.go

```diff
@@ -10,6 +10,29 @@ import (
 	"fmt"
 )
 
+// ListCollaborators list a repository's collaborators
+func (c *Client) ListCollaborators(user, repo string) ([]*User, error) {
```

Comment:
I agree pager is no need currently. But `collaborators := make([]*User, 0, 10)` is really not `collaborators := make([]*User, 10)`.